src/ed25519.rs

```diff
@@ -13,7 +13,7 @@ use serde_bytes::{ByteBuf, Bytes};
 use serde_with::serde_as;
 use signature::{rand_core::OsRng, Signature, Signer, Verifier};
 use std::{
-    fmt::{self, Display},
+    fmt::{self, Display, Debug},
```

Comment:
reorder to `fmt::{self, Debug, Display}` to make rustfmt happy :)

Reply:
Ah yes, will lint